webview/src/experiments/components/App.test.tsx

```diff
@@ -249,6 +253,83 @@ describe('App', () => {
     })
   })
 
+  describe('Sub-rows middle states indicators', () => {
```

Comment:
[F] These were deleted previously. I have adapted the old tests to work on the commit record.